### PR TITLE
add GraphKeys for dynamic_embedding_variable

### PR DESCRIPTION
**Pull request by ccsquare**
# Description

Brief Description of the PR:

Fixes # (issue)

## Type of change

- [ ] Bug fix
- [ ] New Tutorial
- [ ] Updated or additional documentation
- [ ] Additional Testing
- [x] New Feature

# Checklist:

- [x] I've properly [formatted my code according to the guidelines](https://github.com/tensorflow/recommenders-addons/blob/master/CONTRIBUTING.md#coding-style)
    - [ ] By running Black + Flake8
    - [ ] By running pre-commit hooks
- [ ] This PR addresses an already submitted issue for TensorFlow Recommenders Addons
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This PR contains modifications to C++ custom-ops

# How Has This Been Tested?

If you're adding a bugfix or new feature please describe the tests that you ran to verify your changes:
*

### CODE REVIEW

File: tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py [hunk 1]
```diff
@@ -100,6 +100,23 @@ def default_partition_fn(keys, shard_num):
   return ids
 
 
+class DeGraphKeys(object):
```
Comment (rhdong) [inline tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py:103]: de.GraphKeys maybe better, de.DeGraphKeys is a bit redundant

Comment (ccsquare) [reply tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py]: fixed

**Comment by ccsquare** [comment]
@googlebot I fixed it.

File: tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py (hunk 1 above)
Comment (ccsquare) [reply tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py]: @googlebot I fixed it.

**Comment by ccsquare** [comment]
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Frecommenders-addons%2Fpull%2F30) for more info**.

@googlebot I fixed it.

**Comment by ccsquare** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Frecommenders-addons%2Fpull%2F30) for more info**.

@googlebot I fixed it.

**Comment by ccsquare** [comment]
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Frecommenders-addons%2Fpull%2F30) for more info**.

@googlebot I fixed it.

**Review by rhdong — APPROVED** [review]
LGTM

**Pull request merged**